### PR TITLE
Fix #4 (arrays for copyAs)

### LINKED ISSUE #4: copyAs could accept array
From example in readme:

```js
Template.abstract_foo.copyAs('foo');
Template.abstract_foo.copyAs('bar');
```

If we wanted to simplify this slightly, especially in a case where you might have many more, `copyAs` could accept either array or string:

```js
Template.abstract_foo.copyAs(['foo', 'bar']);
```

### PR DESCRIPTION
**Pull request by jgladch**
Pretty straightforward fix here, I believe.

### CODE REVIEW

File: template-extension.js [hunk 1]
```diff
@@ -141,12 +141,23 @@ Template.prototype.inheritsEventsFrom = function (otherTemplateName) {
...
-  var name = parseName(self.viewName);
-  newTemplate.inheritsHelpersFrom(name);
-  newTemplate.inheritsEventsFrom(name);
+    var name = parseName(self.viewName);
+    newTemplate.inheritsHelpersFrom(name);
+    newTemplate.inheritsEventsFrom(name);  
+  };
+
+  //Check if newTemplateName is an array
+  if (Array.isArray(newTemplateName)) {
```
Comment (mitar) [inline template-extension.js]: We already use underscore, so `_.isArray` is probably more portable.

**Comment by mitar** [comment]
LGTM otherwise.

**Comment by mitar** [comment]
Oh, can you also update documentation?

**Comment by aldeed** [comment]
LGTM too after resolution of @mitar's comments.

**Comment by jgladch** [comment]
Now using underscore and updated the README.

**Comment by mitar** [comment]
Huh, you extended the pull request now to include another feature. In general, it is better to not mix things all into one pull request.

**Comment by jgladch** [comment]
Whoops, meant to cut a feature branch for that, my bad. I've rolled-back changes to the scope of this feature.

**Comment by mitar** [comment]
Thanks. Merged.

**Pull request merged**